app/models/status.rb

```diff
@@ -74,6 +75,8 @@ class Status < ApplicationRecord
   scope :not_excluded_by_account, ->(account) { where.not(account_id: account.excluded_from_timeline_account_ids) }
   scope :not_domain_blocked_by_account, ->(account) { account.excluded_from_timeline_domains.blank? ? left_outer_joins(:account) : left_outer_joins(:account).where('accounts.domain IS NULL OR accounts.domain NOT IN (?)', account.excluded_from_timeline_domains) }
 
+  scope :not_local_only, -> { where(local_only: [false, nil]) }
```

Comment:
This is nitpicking (I'm working on more substantial comments, but that's waiting on a deploy...) but I think this can be written as `where.not(local_only: true)`, just as e.g. `not_excluded_by_account` is.

Reply:
that uhhh didn't work
that was what i initially did, but it only found records where `local_only: false`, not `nil`

Reply:
Oh, hmm -- I guess `local_only != true` will interact that way with nulls.

Do we need to permit null in that column?  It seems to me that a status is local-only or not; there's no room for a third value.  If this is the case, we can use `strong_migrations`' guidance for adding columns with a default value: https://github.com/ankane/strong_migrations#adding-a-column-with-a-default-value

Reply:
yeah, i was going to do that but it was a pain and i didn't find that guide

Reply:
I think consensus is that we'll leave it as-is for now.  If we want to constrain the column domain to {true, false} later on, we can do that in a separate migration.